src/Form/Control/ScopeBuilder.php

```diff
@@ -205,8 +205,6 @@ class ScopeBuilder extends Control
             self::OPERATOR_TEXT_DOESNOT_BEGIN_WITH => Condition::OPERATOR_NOT_LIKE,
             self::OPERATOR_TEXT_ENDS_WITH => Condition::OPERATOR_LIKE,
             self::OPERATOR_TEXT_DOESNOT_END_WITH => Condition::OPERATOR_NOT_LIKE,
-            self::OPERATOR_EQUALS => Condition::OPERATOR_EQUALS,
```

Comment:
@georgehristov @ibelar correct to remove?

Reply:
Where is the duplicate?

Reply:
`self::OPERATOR_EQUALS` evaluates to same value as one OPERATOR_TEXT_... :)

Reply:
I believe not:
```
protected const OPERATOR_TEXT_EQUALS = 'is exactly';
...
protected const OPERATOR_EQUALS = 'equals';
```

Reply:
at least `OPERATOR_TEXT_DOESNOT_EQUAL`

Reply:
see https://github.com/atk4/ui/runs/1486496532